boards/b-l072z-lrwan1/Makefile.features

```diff
@@ -1,4 +1,6 @@
 # Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_dma
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
```

Comment:
This line is not needed anymore (see #10637)

Reply:
fixed